volume-specifications/vs-0000-0010.yml

```diff
@@ -0,0 +1,14 @@
+---
+volume: gbenchvol
+options:
+  server.allow-insecure: 'on'
+  server.event-threads: '4'
+  client.event-threads: '4'
+  features.cache-invalidation: 'on'
+  features.cache-invalidation-timeout: '600'
+  performance.stat-prefetch: 'on'
+  performance.cache-invalidation: 'on'
+  performance.md-cache-timeout: '600'
+  network.inode-lru-limit: '90000'
+  cluster.lookup-optimize: 'on'
```

Comment:
I would like group options to also be a part of this, not sure if this is handled here, will check and get back on this. IOW, the comment is this yml should allow setting volume group options as well (I suspect that is the one we would be testing with most often)